users/forms.py

```
@@ -49,14 +54,33 @@ class MultipleSelect(forms.CheckboxSelectMultiple):
+            try:
+                # Get country from form
+                country = self.cleaned_data.get("location_country")
+                
+                # Trigger async task to subscribe user
+                subscribe_user.delay(
+                    email=user.email,
+                    name=user.name,
+                    country=country
+                )
+                logger.info(f"Newsletter subscription task queued for {user.email}")
+            except (ImportError, AttributeError) as e:
```

Ruff (0.14.4)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)